packages/camlhighlight/camlhighlight.5.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+maintainer: "Dario Teixeira <dario.teixeira@nleyten.com>"
+authors: ["Dario Teixeira <dario.teixeira@nleyten.com>"]
+homepage: "http://camlhighlight.forge.ocamlcore.org/"
+bug-reports: "https://github.com/darioteixeira/camlhighlight/issues"
+dev-repo: "https://github.com/darioteixeira/camlhighlight.git"
+license: "GPL-2.0"
+build: [
+  ["./configure" "--prefix" prefix "--docdir" "%{doc}%/camlhighlight"]
+  [make]
+  [make "doc"]
+]
+install: [[make "install"]]
+remove: [["ocamlfind" "remove" "camlhighlight"]]
+depends: [
+  "ocamlbuild" {build}
+  "ocamlfind"
```

Comment:
This could probably be a `{build}` dep unless you use `Findlib` in the package itself. This won't block pub.